CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.1] - 2017-01-10
```

Comment:
I guess technically 1.3.1 could be okay. The API (in theory) should be backwards compatible with 1.3.0, it only adds a new feature.